src/crypto.rs

```diff
@@ -72,7 +78,7 @@ pub fn sign(signing_input: &str, key: &[u8], algorithm: Algorithm) -> Result<Str
     }
 }
 
-/// Encode the claims passed and sign the payload using the algorithm from the header and the key
+/// Encode the header and claims given and sign the payload using the algorithm from the header and the key
```

Comment:
Thoughts on adding examples for each function as well so it's visible from the docs?